package-lock.json

```diff
@@ -1,609 +1,1766 @@
 {
   "name": "antora-lunr",
   "version": "0.8.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Could you please install the dependencies using npm@6 untill we migrate to npm@7?